WebRoot/js/ajax/util/AjxUtil.js

```diff
@@ -69,7 +69,7 @@ AjxUtil.DOMAIN_NAME_FULL_RE = /^[A-Za-z0-9\-]{1,}(\.[A-Za-z0-9\-]{2,}){1,}$/;
 AjxUtil.HOST_NAME_RE = /^[A-Za-z0-9\-]{2,}(\.[A-Za-z0-9\-]{1,})*(\.[A-Za-z0-9\-]{2,})*$/;
 AjxUtil.HOST_NAME_WITH_PORT_RE = /^[A-Za-z0-9\-]{2,}(\.[A-Za-z0-9\-]{2,})*:([0-9])+$/;
 AjxUtil.EMAIL_SHORT_RE = /^[^@\s]+$/;
-AjxUtil.EMAIL_FULL_RE = /^[^@\s]+@[A-Za-z0-9\-]{2,}(\.[A-Za-z0-9\-]{2,})+$/;
+AjxUtil.EMAIL_FULL_RE = /^[^@\s]+@[A-Za-z0-9\-]{1,}(\.[A-Za-z0-9\-]{1,})+$/;
```

Comment:
Do we need to change the length of top domain from `2` to `1`?
Is it reasonable to keep it as it is?
```suggestion
AjxUtil.EMAIL_FULL_RE = /^[^@\s]+@[A-Za-z0-9\-]{1,}(\.[A-Za-z0-9\-]{2,})+$/;
```

Reply:
The above was wrong. Your code change looks good. It will support `a.b.domain.com`.

Reply:
As Kato-san has mentioned above, this regex `/^[^@\s]+@[A-Za-z0-9\-]{1,}(\.[A-Za-z0-9\-]{1,})+$/` allows consequent one-letter subdomain names as well. 　
Example 1: `@a.b.domain.com`
<img width="992" alt="image" src="https://github.com/Zimbra/zm-admin-ajax/assets/12438384/ecc01329-9919-46fa-b6fe-4bf7f40ebbc4">
Example 2: `@123.a.b.com`
<img width="763" alt="image" src="https://github.com/Zimbra/zm-admin-ajax/assets/12438384/578956ee-54c0-4eea-b3d8-0b731e1bcd5e">
The following examples are slightly unrealistic, but RFC-wise we are allowed such sub/top-level/domain names (see RFC 1034 Section 3.5, RFC-1123 section 2.1 as well):
Example 3: `@abc.1.2.3`
<img width="957" alt="image" src="https://github.com/Zimbra/zm-admin-ajax/assets/12438384/c4c03fe3-25c8-432d-b8c8-ab64c82ea4df">